### PR TITLE
Client method to get ss version

### PR DESCRIPTION
**Pull request by bharat-thotakura**
Added a method to _SuperstaQClient to retrieve the SuperstaQ version per issue SupertechLabs/superstaq-client#381

### CODE REVIEW

File: general_superstaq/superstaq_client.py [hunk 1]
```diff
@@ -107,6 +107,18 @@ def request() -> requests.Response:
 
         return self._make_request(request).json()
 
+    def get_superstaq_version(self) -> Dict[str, Optional[str]]:
+        """Gets SuperstaQ version from response header
+
+        Returns:
+            SuperstaQ version in dict
```
Comment (paaige) [inline general_superstaq/superstaq_client.py:114]: nit: 'A dict containing the current SuperstaQ version.'

**Review by paaige — APPROVED** [review]

File: general_superstaq/superstaq_client.py (hunk 1 above)
Comment (bharat-thotakura) [reply general_superstaq/superstaq_client.py]: Updated in commit (0fad825)

Comment (paaige) [reply general_superstaq/superstaq_client.py]: All that's left is to merge with main and this PR is good to go!

**Comment by richrines1** [comment]
@bharat-thotakura do you mind migrating this branch to https://github.com/SupertechLabs/client-superstaq and opening a pr there (and then closing this one)? (lmk if you have trouble migrating)

**Comment by bharat-thotakura** [comment]
> @bharat-thotakura do you mind migrating this branch to https://github.com/SupertechLabs/client-superstaq and opening a pr there (and then closing this one)? (lmk if you have trouble migrating)

Sure thing

**Comment by vtomole** [comment]
@bharat-thotakura I'm going to archive this repo: You should have the source for this PR in your local branch so should be straightforward to copy-paste it to `client-superstaq`

**Pull request closed without merging**